lib/index.ts

```diff
@@ -19,19 +20,25 @@ const internals = {
   LocalVideoTrack: require('./media/track/es5').LocalVideoTrack
 };
 
+interface CreateLocalTrackOptionsInternal extends CreateLocalTrackOptions {
+  createLocalTracks: typeof createLocalTracks
+}
+
 function connect(token: string, options?: ConnectOptions): Promise<Room> {
+  const internalOptions = options as ConnectOptionsInternal;
+  internalOptions.createLocalTracks = createLocalTracks;
```

Comment:
options is optional...so we cannot dereference it directly.
```suggestion
  const internalOptions = {
     createLocalTracks,
     ...options
  }
```

Reply:
This makes sense. But this would also mean we would no longer need `ConnectOptionsInternal`?

Reply:
yes, `ConnectOptionsInternal` may not be necessary since connect is not yet converted to typescript.